crest/Assets/Crest/Crest/Scripts/LodData/RegisterSeaFloorDepthInput.cs

```diff
@@ -27,4 +27,31 @@ protected override void OnEnable()
             }
         }
     }
+
+
+    /// <summary>
+    /// Tags this object as an ocean depth provider with geometry shader support (for advanced users).
+    ///
+    /// This is similar to RegisterSeaFloorDepthInput, but allows geometry depth to be rendered to all lod levels in a
+    /// single dispatch. The Ocean Depth Cache supports this mode of rendering, so use it's implementation as a
+    /// reference on how to do this.
+    /// </summary>
+    public class RegisterSeaFloorDepthInputGeometry : RegisterLodDataInput<LodDataMgrSeaFloorDepth>
```

Comment:
i asked on discord but guess it makes sense to register here as well - could the presence of a GS be automatically detected from the shader, so the user doesn't have to configure and maintain a separate "Register" script type?